src/gateways/gov-notify.js

```diff
@@ -1,19 +1,25 @@
 import { DefaultGateway } from '../gateways/default-gateway';
 
 export class GovNotifyGateway extends DefaultGateway {
-    async sendText(phoneNumber, message) {
-        const response = await this.postToUrl(`gov-notify/${message}?phoneNumber=${phoneNumber}`);
+    async sendText(phoneNumber, templateAlias, templateParams) {
```

Comment:
Have I understood this correctly, we've previously been sending texts via Gov Notify that were raw text and not tied to a template?

Reply:
I'm also just as confused about this.
Looking into the tests in the file `src/gateways/gov-notify.test.js` bellow, it would seem that is the case. I think that the templates that were used by `send single text or email` functionality don't have any personalisation in them. That's how it seemed from the one template that I saw, but I can't tell anything about others if there are any.